hedera-node/hedera-config/src/testFixtures/java/com/hedera/node/config/testfixtures/HederaTestConfigBuilder.java

```diff
@@ -125,7 +124,6 @@ public static TestConfigBuilder create() {
                 .withConfigDataType(MetricsConfig.class)
                 .withConfigDataType(PrometheusConfig.class)
                 .withConfigDataType(PlatformStatusConfig.class)
-                .withConfigDataType(FCHashMapConfig.class)
```

Comment:
Why do you remove that class from services?

Reply:
I removed it from everywhere. Settings for `FCHashMap` became static. I did this for a few reasons:

- I added these settings a few years ago and we have never used anything except for the defaults
- `FCHashMap` is no longer used in production (replaced by `VirtualMap`)
- if this configuration remained it would have to be static config, which I am on a crusade to eliminate :)